Storage/SQL/BrowserSchema.swift

```diff
@@ -232,155 +231,180 @@ open class BrowserSchema: Schema {
+        """
+
+    let syncCommandsTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableSyncCommands) (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            client_guid TEXT NOT NULL,
+            value TEXT NOT NULL
+        )
+        """
+
+    let clientsTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableClients) (
+            guid TEXT PRIMARY KEY,
+            name TEXT NOT NULL,
+            modified INTEGER NOT NULL,
+            type TEXT,
+            formfactor TEXT,
+            os TEXT,
+            version TEXT,
+            fxaDeviceId TEXT
+        )
+        """
+
+    let tabsTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableTabs) (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            client_guid TEXT REFERENCES clients(guid) ON DELETE CASCADE,
+            url TEXT NOT NULL,
+            title TEXT,
+            history TEXT,
+            last_used INTEGER
+        )
+        """
+
+    let activityStreamBlocklistCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableActivityStreamBlocklist) (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            url TEXT NOT NULL UNIQUE,
+            created_at DATETIME DEFAULT CURRENT_TIMESTAMP
+        )
+        """
+
+    let pageMetadataCreate = """
+        CREATE TABLE IF NOT EXISTS \(TablePageMetadata) (
+            id INTEGER PRIMARY KEY,
+            cache_key LONGVARCHAR UNIQUE,
+            site_url TEXT,
+            media_url LONGVARCHAR,
+            title TEXT,
+            type VARCHAR(32),
+            description TEXT,
+            provider_name TEXT,
+            created_at DATETIME DEFAULT CURRENT_TIMESTAMP,
+            expired_at LONG
+        )
+        """
+
+    let highlightsCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableHighlights) (
+            historyID INTEGER PRIMARY KEY,
+            cache_key LONGVARCHAR,
+            url TEXT,
+            title TEXT,
+            guid TEXT,
+            visitCount INTEGER,
+            visitDate DATETIME,
+            is_bookmarked INTEGER
+        )
+        """
 
     let indexPageMetadataCacheKeyCreate =
-    "CREATE UNIQUE INDEX IF NOT EXISTS \(IndexPageMetadataCacheKey) ON page_metadata (cache_key)"
+        "CREATE UNIQUE INDEX IF NOT EXISTS \(IndexPageMetadataCacheKey) ON page_metadata (cache_key)"
 
     let indexPageMetadataSiteURLCreate =
-    "CREATE UNIQUE INDEX IF NOT EXISTS \(IndexPageMetadataSiteURL) ON page_metadata (site_url)"
+        "CREATE UNIQUE INDEX IF NOT EXISTS \(IndexPageMetadataSiteURL) ON page_metadata (site_url)"
 
     let iconColumns = ", faviconID INTEGER REFERENCES \(TableFavicons)(id) ON DELETE SET NULL"
     let mirrorColumns = ", is_overridden TINYINT NOT NULL DEFAULT 0"
 
-    let serverColumns = ", server_modified INTEGER NOT NULL" +    // Milliseconds.
-                        ", hasDupe TINYINT NOT NULL DEFAULT 0"    // Boolean, 0 (false) if deleted.
+    let serverColumns = """
+        -- Milliseconds.
```

Comment:
Is there a reason you chose to do it this way rather than

```sql
, server_modified INTEGER NOT NULL     -- Milliseconds.
```

Reply:
Yes. In cases where we embed one bit of SQL within another, to avoid a trailing `-- comment` from early-terminating another part of SQL.

Reply:
I'm glad I asked! I hadn't thought of that. Good enough for me.